src/client/Client.cc

```diff
@@ -2966,10 +2966,15 @@ Dentry* Client::link(Dir *dir, const string& name, Inode *in, Dentry *dn)
     dn->dir = dir;
     dir->dentries[dn->name] = dn;
     lru.lru_insert_mid(dn);    // mid or top?
+    if (!in)
```

Comment:
zheng, i saw the comment above this function, and it says this function does not expect a NULL 'in', how do you think of that ?

Reply:
thanks, so the commet is not correct any more